src/web/mjs/connectors/Comikey.mjs

```diff
@@ -0,0 +1,90 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class ComiKey extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'comikey';
+        super.label = 'Comikey';
+        this.tags = ['manga', 'webtoon', 'english'];
+        this.url = 'https://comikey.com';
+        this.chapterUrl = 'https://relay-us.epub.rocks/consumer/COMIKEY/series/';
+    }
+
+    _getMangaListFromPages(page) {
+        page = page || 1;
+        let uri = new URL('/sapi/comics', this.url);
+        uri.searchParams.set('page', page);
+        let request = new Request(uri.href, this.requestOptions);
+        return this.fetchJSON(request)
+            .then(data => {
+                let mangaList = data.results.map(manga => {
+                    return {
+                        id: manga.id + ',' + manga.e4pid,
+                        title: manga.name
+                    };
+                });
+                if (mangaList.length == 25) {
```

Comment:
Does this work when the last page has exactly `25` mangas, or will this become an infinite recursive call?

Reply:
As of now, there are exactly 25 responses per page. By checking the count, it should break any case of infinite loop. Technically it throws 404 when page not available.